Makefile

```diff
@@ -0,0 +1,63 @@
+NAME=cimgui-go Code Generator
+GOCMD=LC_ALL=C go
+TIMEOUT=5
+
+# go tools
+export PATH := ./bin:$(PATH)
+export GO111MODULE := on
+export GOPROXY = https://proxy.golang.org,direct
+
+.PHONY: all setup cimgui cimplot generate
+
+all: generate
+
+## setup: pulls down dependencies; builds codegen programm
+setup:
+	$(GOCMD) build -o ./cmd/codegen/build/codegen ./cmd/codegen
```

Comment:
Do you really need a "setup" step? Can't it be just "go run"?

Reply:
idk, it was this way in justfile so I didn't change anything so far.

Reply:
I think, that `go run` is a good idea, but i'd leave the step and put here `go get -d ./...` for now
an maybe in th efuture, if we'll need to use `go generate` or something like that, we may put it there.